### PR TITLE
Issue 193 related policies 

### PR DESCRIPTION
**Pull request by felizj17**
Contributes to #193 

## What did you do?
Added the related policies to the details accordion. Created a RelatedPolicyDetails component and test for it.  Also imported the filtermenu.scss file to the FilterMenu.js file 
## Why did you do it?
it was needed.
## How have you tested it?
Wrote test for it and viewed the page
## Were docs updated if needed?

- [ ] No
- [ ] Yes
- [x] N/A

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new console logs
- [ ] Fixes entire issue
- [x] Partial fix for issue

### CODE REVIEW

File: client/src/policyDetail/DetailsAccordion.js [hunk 1]
```diff
@@ -48,8 +49,17 @@ const DetailsAccordion = ({
...
         title={intl.formatMessage(messages.detailsItemRelatedPolicies)}
       >
-        {/*<p>{relatedPolicies}</p>*/}
-        <p>TODO: handle array of related policies</p>
+        {relatedPolicies.length ? (
+          relatedPolicies.map((relatedPolicy, index) => (
+            <RelatedPolicyDetails
+              key={`related-${relatedPolicy}-${index}`}
+              policy={relatedPolicy.title}
+              id={relatedPolicy.id}
```
[the remark this answers is not in the document]
Comment (markstur) [reply client/src/policyDetail/DetailsAccordion.js:57]: I think the fix is to change the prop type to number (see below)

File: client/src/policyDetail/RelatedPolicyDetails.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const RelatedPolicyDetails = ({ policy, id }) => {
+  return (
+    <p data-testid={'relatedPolicy'}>
+      <a href={`./${id}`}>{policy}</a>
+    </p>
+  )
+}
+RelatedPolicyDetails.propTypes = {
+  policy: PropTypes.string,
+  id: PropTypes.string,
```
Comment (markstur) [inline client/src/policyDetail/RelatedPolicyDetails.js:13]: id: PropTypes.number. will fix the string/number problem (see other comment).
Not sure if the fix should be here or above, but I think here.

**Review by markstur — CHANGES_REQUESTED** [review]
There is a string/number mismatch that should be fixed.  Some other minor things (see inline comments) that you should fix while fixing that one problem.

**Comment by felizj17** [comment]
I'll take care of it today, been a little busy

**Review by ed-snodgrass — APPROVED** [review]
Looks good to me. All comments were addressed. Tests all pass, no linting issues. And it works :)

**Review by markstur — APPROVED** [review]
Thanks!

**Pull request merged**